Remotion/Web/Core/Res/Themes/NovaViso/HTML/WebButton.css

```diff
@@ -2,105 +2,54 @@
 
 button.webButton
 {
-  background-color: Transparent;
-  cursor: hand;
+  background-color: transparent;
+  border: 1px solid var(--color-primary);
+  border-radius: 2px;
+  padding: 0.6em;
+  box-sizing: border-box;
   cursor: pointer;
-  text-decoration: none !important;
-  border: none;
-  position: relative;
-  display: inline-block;
-  vertical-align: middle;
-  padding: 0px 2px;
-  -moz-padding-start: 0;
-  -moz-padding-end: 0;
-  outline: none;
-}
-
-button.webButton::-moz-focus-inner
-{ 
-  border: 0;
 }
 
-button.webButton span.buttonBody
+button.webButton:hover,
+button.webButton:focus
 {
-  border: solid 1px #AAAAAA;
-  font-weight: bold;
-  background-image: url(../Image/ButtonBackground.gif);
-  background-color: #E6E5E7;
-  background-position: center top;
-  background-repeat: repeat-x;
-  
-  display: block;
-  position: relative;
-  top: 0px;
-  left: -2px;
-  
-  width: 100%;
-  white-space: nowrap;
-
-  padding-top: 2px;
-  padding-bottom: 2px;
-  padding-left: 1px;
-  padding-right: 1px;
+  background-color: var(--color-highlight);
 }
 
-button.webButton span.buttonBody:first-child::before
+button.webButton.primary
 {
-  content: "";
-  height: 1.25em;
-  width: 0;
-  display: inline-block;
-  vertical-align: text-bottom;
-}
-
-button.webButton span.buttonBody img
-{
-  vertical-align: bottom;
+  background-color: var(--color-primary);
+  color: white;
   border: none;
 }
 
-button.webButton:focus span.buttonBody,
-button.webButton:hover span.buttonBody
+button.webButton.primary:hover,
+button.webButton.primary:focus
 {
-  background-image: url(../Image/ButtonBackgroundHover.gif);
-  background-color: #DEECFC;
-  border: 1px solid #7CB2E6;
+  background-color: var(--color-primary-highlight);
 }
 
-button.webButton:focus span.buttonBody
+button.webButton.secondary
 {
-  text-decoration: underline;
+  border: none;
 }
 
-button.webButton:active span.buttonBody
+button.webButton[disabled]
 {
-  background-image: url(../Image/ButtonBackgroundActive.gif);
-  background-color: #C5E0FC;
-  border: 1px solid #569CE0;
+  background-color: transparent;
+  color: var(--color-standard-border);
+  border: 1px solid var(--color-standard-border);
+  cursor: unset;
 }
 
-button.webButton[disabled] span.buttonBody,
-button.webButton[disabled]:hover span.buttonBody,
-button.webButton[disabled]:active span.buttonBody
+button.webButton span.buttonBody
 {
-  color: Gray;
-  border: solid 1px #BCBCBC;
-  background: none;
-  background-color: #EFEFF0;
-  cursor: default;
+  display: flex;
+  align-items: center;
```

Comment:
The vertical alignment is more complicated. only the on-the-line portion of the text should be vertically centered. I do not know if we have a baseline option with flexgrif or if we need to use other means.

Reply:
Well the on-the-line portion of the text is vertically centered now, as well as the image, or am I missing something?

Reply:
![image](https://user-images.githubusercontent.com/6972548/129588191-a4b35645-9cf0-4c2f-b833-86c99e7f6b80.png)

It doesn't look that way to me?

Reply:
My (not so scientific) measurements:
![image](https://user-images.githubusercontent.com/40357550/129699078-2558085f-31ad-4c61-9558-85283b37ae3f.png)

Pretty centered imo, even though its not perfect.